core/src/io/anuke/mindustry/input/DesktopInput.java

```diff
@@ -429,6 +429,17 @@ void pollInput(){
 
             mode = none;
         }
+
+        if(Core.input.keyTap(Binding.toggle_power_lines)){
+            if(Core.settings.getInt("lasersopacity") == 0){
+                Core.settings.put("lasersopacity", Core.settings.getInt("preferredlaseropacity"));
```

Comment:
I would change this to `Core.settings.getInt("preferredlaseropacity", 100)` to account for the slim chance that preferred laser opacity is missing.